app/controllers/dashboard/base_controller.rb

```diff
@@ -4,4 +4,14 @@ class Dashboard::BaseController < ApplicationController
   def current_account
     current_user.account
   end
+
+  private
+
+  def authorize_admin!
+    deny_access! unless current_user.is_admin?
+  end
+
+  def deny_access!
+    head(:unauthorized)
```

Comment:
Should be redirected instead of just empty response coz it's not in API, this one is html request.